src/xrScriptEngine/ScriptEngineScript.cpp

```diff
@@ -164,4 +194,14 @@ SCRIPT_EXPORT(CScriptEngine, (),
         def("editor", &is_editor),
         def("user_name", &user_name)
     ];
+
+    module(luaState, "profiler")
+    [
+        def("isActive", &isProfilerActive),
```

Comment:
Using lambdas is preferred:
```suggestion
        def("isActive", +[]()
        {
            GEnv.ScriptEngine->m_profiler->isActive();
        }),
```
The `+` before `[]` is mandatory for successful compilation.